azurelinuxagent/pa/rdma/suse.py

```diff
@@ -25,7 +25,7 @@
 
 class SUSERDMAHandler(RDMAHandler):
 
-    def install_driver(self):  # pylint: disable=R0912,R1710
+    def install_driver(self):  # pylint: disable=R1710
```

Comment:
As a note to any potential future reviewers, the remaining warning suppression `R1710=inconsistent-return-statements` is _not_ style related. It marks this function as having `return` and `return True` statements in the same function.